.env

```diff
@@ -0,0 +1,54 @@
+# Basenet settings
```

Comment:
I'd move this somewhere to `.github/testcases-env`. It can then be copied to any appropriate location in workflows, but `~/.env` just doesn't look right.

Reply:
> Another question is do we need it at all, it's about the same as the one we have in `neofs-testcases` and we have to fetch/use this repo anyway.

With the config files we will be able to change the configuration for each component and each branch.
I plan to have such configuration files in every repos.
It will be possible to make a flexible test system where all the dependency information will be along with the code.

Reply:
You can and should do it dynamically when and where it's needed. Grab a "reference" copy from testcases repo, stuff some versions that we need to test into it and go.